recipe/meta.yaml

```diff
@@ -11,6 +11,7 @@ source:
 build:
   number: 0
   script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py2k]
```

Comment:
cc @jcapriot. I believe this is how we skip python 2.7 builds (see conda-forge/conda-smithy#1110)